charts/camunda-platform/templates/camunda/constraints.tpl

```diff
@@ -0,0 +1,16 @@
+{{/*
+A template to handle constraints.
+*/}}
+
+{{/*
+Show a message if Optimize is enabled but identity is disabled
```

Comment:
Let's add a link to the env reference here:
https://docs.camunda.io/docs/reference/supported-environments/#camunda-8-self-managed